config.xml

```diff
@@ -8,6 +8,7 @@
     <content src="index.html" />
     <plugin name="cordova-plugin-whitelist" spec="1" />
     <plugin name="cordova-clipboard" spec="^1.3.0" />
+    <plugin name="cordova-plugin-x-socialsharing" spec="^6.0.3" />
```

Comment:
Version `^6.0.0` includes support for Android X, build on Android is failing because of that.
`This project uses AndroidX dependencies, but the 'android.useAndroidX' property is not enabled. Set this property to true in the gradle.properties file and retry`